test/signalrclient-e2e-tests/url.h

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#pragma once
+
+extern utility::string_t url;
```

Comment:
As @BrennanConroy said - the definition should not be in a header and then this extern would go to test_utils I think.